fedimint-server/Cargo.toml

```diff
@@ -44,6 +45,11 @@ tokio-stream = "0.1.11"
 tokio-rustls = "0.23.4"
 tokio-util = { version = "0.7.4", features = [ "codec" ] }
 tracing-subscriber = { version = "0.3.16", features = [ "env-filter" ] }
+aleph-bft = { version = "0.20.5", default-features = false }
+bitcoin_30 = { package = "bitcoin", version = "0.30.0" }
```

Comment:
Do we really really need it? :D

Reply:
We need bitcoin_30 for calculating merle roots of blocks